core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -53,8 +53,6 @@
  *   <li>{@code p1 AND p2} is strong in [p1, p2] (definitely null if either p1
  *   is null or p2 is null)
  *   <li>{@code p1 OR p2} is strong if p1 and p2 are strong
- *   <li>{@code c1 = 1 OR c2 IS NULL} is strong in [c1] (definitely null if c1
```

Comment:
LGTM.
By the way, can you also correct the spelling error of the idea prompt, changing `with regards to` to `with regard to`.

https://github.com/apache/calcite/blob/c546d4a59cf1cb20e6ec2a8981decf179d9a6b50/core/src/main/java/org/apache/calcite/plan/Strong.java#L41

Reply:
Sure.